packages/waku/src/lib/middleware/rsc.ts

```diff
@@ -193,6 +190,13 @@ export function rsc<
         return;
       }
     } else {
+      if (pathStr === '/') {
+        const htmlStr = await getHtmlStr(pathStr);
+        res.setHeader('Content-Type', 'text/html');
+        res.setStatus(200);
+        endStream(res.stream, htmlStr!);
+        return;
+      }
```

Comment:
Can you explain why you need this?

Reply:
In dev mode, vite expects index.html to be in / and not in /src/, since we don't want to move it to /, we need to handle it manually like the way we handle it in ssr.

Reply:
And the obvious solution (which is a breaking change) that's aligned with the PR, is to move index.html to / rather than having it in /src/

Any other idea?

Reply:
I'm not against to it. Can you open a new PR for that idea?

Reply:
> we'd have /src/foo/index.html at some point?

Yes, we have `/foo` and now we have such an e2e test, so the test fails.